src/box/lua/tuple_format.c

```diff
@@ -98,12 +142,24 @@ box_lua_tuple_format_init(struct lua_State *L)
 	};
 	luaL_register_type(L, tuple_formatlib_name, lbox_tuple_format_meta);
 
+	const struct luaL_Reg lbox_tuple_formatlib[] = {
+		{"is", lbox_tuple_format_is},
+		{NULL, NULL}
+	};
+	luaL_findtable(L, LUA_GLOBALSINDEX, tuple_formatlib_name, 0);
+	luaL_setfuncs(L, lbox_tuple_formatlib, 0);
+	lua_pop(L, 1);
+
 	struct luaL_Reg boxlib_internal[] = {
 		{"new_external_tuple_format", lbox_new_external_tuple_format},
 		{NULL, NULL}
 	};
 
-	luaL_findtable(L, LUA_GLOBALSINDEX, "box.internal", 0);
+	luaL_findtable(L, LUA_GLOBALSINDEX, "box.internal", 1);
 	luaL_setfuncs(L, boxlib_internal, 0);
+	lua_createtable(L, 0, 1);
+	lua_pushcfunction(L, lbox_tuple_format_new);
+	lua_setfield(L, -2, "new");
+	lua_setfield(L, -2, "tuple_format");
```

Comment:
Nit: Can't you just add this function to `lbox_tuple_formatlib` instead of pushing it explicitly?

Reply:
Fixed.